cmd/describe/ingress/cmd.go

```diff
@@ -0,0 +1,185 @@
+package ingress
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strconv"
+	"strings"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/helper"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+// Regular expression to used to make sure that the identifier given by the
+// user is safe and that it there is no risk of SQL injection:
+var ingressKeyRE = regexp.MustCompile(`^[a-z0-9]{3,5}$`)
+
+var Cmd = &cobra.Command{
+	Use:     "ingress",
+	Short:   "Show details of the specified ingress within cluster",
+	Example: `rosa describe ingress <ingress_id> -c mycluster`,
+	Run:     run,
+	Args: func(_ *cobra.Command, argv []string) error {
+		if len(argv) != 1 {
+			return fmt.Errorf(
+				"Expected exactly one command line parameter containing the id of the ingress",
+			)
+		}
+		return nil
+	},
+}
+
+func init() {
+	ocm.AddClusterFlag(Cmd)
+	output.AddFlag(Cmd)
+}
+
+func run(_ *cobra.Command, argv []string) {
+	r := rosa.NewRuntime().WithOCM()
+	defer r.Cleanup()
+
+	ingressID := argv[0]
+	if !ingressKeyRE.MatchString(ingressID) {
+		r.Reporter.Errorf(
+			"Ingress  identifier '%s' isn't valid: it must contain only letters or digits",
+			ingressID,
+		)
+		os.Exit(1)
+	}
+
+	clusterKey := r.GetClusterKey()
+	cluster := r.FetchCluster()
+
+	// Try to find the ingress:
+	r.Reporter.Debugf("Loading ingresses for cluster '%s'", clusterKey)
```

Comment:
@gdbranco How about collapsing this into a `GetIngress(clusterId string, ingressId string)` method on `OCMClient`?